<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIPanel</class>
 <widget class="QWidget" name="GUIPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUIPanel</string>
  </property>
  <widget class="QPushButton" name="runMQTTButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Inicio</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>331</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Detenido</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>64</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Estado:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>MQTT Broker</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>37</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Topic</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leHost">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>164</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>localhost</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="topic">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>35</y>
     <width>164</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>/rpi/btntopic</string>
   </property>
  </widget>
  <widget class="Led" name="led">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Color Led component</string>
   </property>
   <property name="whatsThis">
    <string>Led indicator</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="color">
    <color>
     <red>85</red>
     <green>170</green>
     <blue>255</blue>
    </color>
   </property>
  </widget>
  <widget class="Led" name="led_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Color Led component</string>
   </property>
   <property name="whatsThis">
    <string>Led indicator</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="color">
    <color>
     <red>85</red>
     <green>170</green>
     <blue>255</blue>
    </color>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>LEFT_BUTTON</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RIGHT_BUTTON</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="leftNumber">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>92</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="mode">
    <enum>QLCDNumber::Dec</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="rightNumber">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="mode">
    <enum>QLCDNumber::Dec</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Led</class>
   <extends>QWidget</extends>
   <header>analogwidgets/led.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>runMQTTButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
